contents/docs/integrations/js-integration.md

```diff
@@ -126,7 +126,11 @@ This will automatically send the current URL.
 
 ### Super Properties 
 
-You can register 'global' properties for a user, that will persist between sessions. Every `capture` call will include these properties. These are not to be confused with the `people.set` call, which stores properties against the User object (rather than against every event).
+Super Properties are properties that are associated with every event by a given user, while also persisting across sessions. Every `capture` call will include these properties i.e. the properties will be passed with any event, be it a `$pageview`, a button click, or anything else. 
```

Comment:
Everything below reads well for me. The only thing adding a bit of confusion is the "by a given user" bit in the first sentence. This makes it seem like the properties are tied to a user... before reading the clarification below that they are per-event, not per-user.

Perhaps a simplification like this makes it clearer:

> Super Properties are properties that are set once and then sent with every `capture` call, be it a `$pageview`, an autocaptured button click, or anything else. 

... and then somewhere in the bottom (near register_once?), add that once set, super properties persist across sessions... until `posthog.reset()` is called.

Reply:
Makes sense to me!